entry_types/scrolled/package/src/frontend/MediaPlayerControls.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {usePlayerState} from './MediaPlayer';
+import {ControlBar} from './playerControls/ControlBar'
+
+export function MediaPlayerControls(props) {
+  const [playerState, playerActions] = usePlayerState();
```

Comment:
These should be passed in as props to allow sharing them with a `MediaPlayer`. `InlineVideo` above already uses the hook and will need to render `MediaPlayerControls` in the next step.